prow/cmd/mkpj/main.go

```diff
@@ -159,6 +190,7 @@ func gatherOptions() options {
 	fs.IntVar(&o.pullNumber, "pull-number", 0, "Git pull number under test")
 	fs.StringVar(&o.pullSha, "pull-sha", "", "Git pull SHA under test")
 	fs.StringVar(&o.pullAuthor, "pull-author", "", "Git pull author under test")
+	fs.StringVar(&o.pullRequestChanges, "pull-changes", "", "Git pull changed files, comma-separated")
```

Comment:
You could use `prow/flagutil/strings.go#Strings` and not need to implement parsing

Reply:
Thank you for the review. I will update the PR.